react-frontend/src/components/LoginComponent.js

```diff
@@ -0,0 +1,112 @@
+import React, { useState } from "react";
+import "bootstrap/dist/css/bootstrap.min.css";
+import * as yup from "yup";
+import {
+  Col,
+  Form,
+  Input,
+  Button,
+  Container,
+  FormGroup,
+  Label,
+  InputGroup,
+} from "reactstrap";
```

Comment:
All of us need to use common package. We are already using `react-bootstrap`. Check `package.json` file